src/b3d/chisight/gen3d/pixel_kernels/pixel_color_kernels.py

```diff
@@ -26,32 +26,39 @@ class PixelColorDistribution(genjax.ExactDensity):
     An abstract class that defines the common interface for pixel color kernels.
 
     Distribuiton args:
-    - latent_rgb
-    - rgb_scale
+    - latent_color
+    - color_scale
 
     Support:
     - An RGB value in [0, 1]^3.
     """
 
     @abstractmethod
     def sample(
-        self, key: PRNGKey, latent_color: FloatArray, *args, **kwargs
```

Comment:
I removed `*args` and `**kwargs` throughout this to emphasize that the `PixelColorKernel` interface exactly has signature `(latent_color, color_scale)` with no additional args and kwargs.